test_command_line_tool.d

```diff
@@ -12,13 +12,35 @@ int main(string[] args)
     try { return tryMain(args); }
     catch(SilentException) { return 1; }
 }
+
 int tryMain(string[] args)
 {
-    foreach (entry; dirEntries("test", "*.har", SpanMode.shallow))
+    // TODO: move to std.path
+    version(Windows)
+        string exeExtention=".exe";
+    else
+        string exeExtention;
+
+    auto rootDir=__FILE_FULL_PATH__.dirName;
+    auto outDir = rootDir.buildPath("out");
+    auto harExe = outDir.buildPath("har"~exeExtention);
```

Comment:
space between the `~` operator (consistent with other D projects)

Reply:
done